tests/unit/components/sl-unable-test.js

```diff
@@ -53,22 +53,111 @@ test( 'Does not render content', function( assert ) {
+        )
+    });
+
+    assert.equal(
+        Ember.$.trim( this.$().text() ),
+        'Should render'
+    );
+});
+
 test( 'isUnable() calls isUnable() on the behavior service', function( assert ) {
     let component = this.subject({
         behaviorService: behaviorService,
-        behavior: 'the_behavior',
-        provider: 'the_provider'
+        activity: 'anActivity',
+        resource: 'aResource',
+        possible: true
     });
 
     this.render();
 
-    assert.equal(
-        behaviorService.get( 'behavior' ),
-        'the_behavior'
+    assert.ok(
+        behaviorService.isUnable.withArgs( 'anActivity', 'aResource', true ).calledOnce,
+        'isUnable() was called with the correct params'
     );
+});
 
-    assert.equal(
-        behaviorService.get( 'provider' ),
-        'the_provider'
+test( 'Accepts a function as the third parameter', function( assert ) {
+    let component = this.subject({
+        behaviorService: behaviorService,
+        activity: 'anActivity',
+        resource: 'aResource',
+        possible: () => false
+    });
+
+    this.render();
+
+    assert.ok(
+        behaviorService.isUnable.withArgs( 'anActivity', 'aResource', false ).calledOnce,
+        'isAble() was called with the correct params'
+    );
+});
+
+test( 'Assert is thrown when `possible` is a function not returning a Boolean', function( assert ) {
+    let component = this.subject({
+        behaviorService: behaviorService,
+        activity: 'anActivity',
+        resource: 'aResource',
+        possible: () => 'not a boolean'
+    });
+
+    assert.throws(
+        this.render,
+        'Assert is thrown'
+    );
+});
+
+test( 'Is responsive to changes in the behavior data on the service', function( assert ) {
+    let component = this.subject({
+        behaviorService: behaviorService,
+        activity: 'anActivity',
+        resource: 'aResource'
+    });
+
+    this.render();
+
+    Ember.run( () => {
+        behaviorService.set( 'behaviors', {
+            'aResource': {
+                'anActivity': false
+            }
+        });
+    });
+
+    assert.ok(
+        behaviorService.isUnable.withArgs( 'anActivity', 'aResource', true ).calledTwice,
+        'isUnable() is called twice'
+    );
+});
+
+test( 'Is responsive to changes to the `possible` parameter', function( assert ) {
+    let component = this.subject({
+        behaviorService: behaviorService,
+        activity: 'anActivity',
+        resource: 'aResource'
+    });
+
+    this.render();
+
+    assert.ok(
+        behaviorService.isUnable.withArgs( 'anActivity', 'aResource', true ).calledOnce,
+        'isUnable() is called with `true` as a third param'
+    );
+
+    Ember.run( function() {
+        component.set( 'possible', false );
+    });
+
+    assert.ok(
+        behaviorService.isUnable.withArgs( 'anActivity', 'aResource', false ).calledOnce,
+        'isUnable() is called with `false` as a third param'
     );
 });
```

Comment:
As discussed, this test will be combined with the sl-able component test and their content actually applied to the component mixin test, leaving this one to only test that `isUnable()` is called on the service.